packages/atomic/src/components/facets/atomic-date-facet/atomic-date-facet.tsx

```diff
@@ -35,14 +48,32 @@ export class AtomicDateFacet implements InitializableComponent, BaseFacetState {
 
   @State() public isExpanded = false;
   @Prop({mutable: true, reflect: true}) public facetId = '';
+  /**
+   * Specifies the index field whose values the facet should use
+   */
   @Prop() public field = '';
```

Comment:
There's a case to be made to make this `@date` the default... but it's not 100% mandatory.